about.html

```diff
@@ -0,0 +1,150 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=BIZ+UDPGothic:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <!-- <link rel="stylesheet" href="./css/home.css" /> -->
+    <link rel="stylesheet" href="./css/about.css" />
+    <title>Genuine Charity | about</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar">
+        <div class="top-header">
+          <ul class="top-header-navbar">
+            <li class="nav-item">
+              <a href="#" class="nav-link"
+                ><i class="fa-brands fa-facebook-f"></i
+              ></a>
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-link"
+                ><i class="fa-brands fa-twitter"></i
+              ></a>
+            </li>
+            <li class="nav-item"><a href="#" class="nav-link">English</a></li>
+            <li class="nav-item"><a href="#" class="nav-link">My Page</a></li>
+            <li class="nav-item"><a href="#" class="nav-link">Logout</a></li>
+          </ul>
+        </div>
+        <div class="header-bottom">
+          <div class="logo"><a href="./index.html">My Logo</a></div>
+          <div class="nav-right">
+            <ul class="main-nav">
+              <li class="nav-item">
+                <a href="./index.html" class="nav-link">Home</a>
+              </li>
+              <li class="nav-item">
+                <a href="#program" class="nav-link">Program</a>
+              </li>
+              <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+              <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+              <li class="nav-item">
+                <a href="#sponsor" class="nav-link">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link campain">CG Campain</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <main class="about-wrapper">
+      <section class="about">
+        <div class="about-us">
+          <span class="about-quote"
+            >“Our highest deeds come from helping the lowest people.”</span
+          >
+          <h1 class="about-goal">
+            “Overcoming poverty is not a gesture of charity. It is an act of
+            justice. It is the protection of a fundamental human right, the
+            right to dignity and a decent life.”
```

Comment:
I don't get how my element are not centered because that exactly what I'm saying in my computer. sure thing You ca.y want them to fit the same space as the one mentioned in the template because as you can see, the one in the template the only have few words (sentences);
:-1: @brytebee

Reply:
I don't get how my element are not centered because that exactly what I'm saying in my computer. sure thing You ca.y want them to fit the same space as the one mentioned in the template because as you can see, the one in the template the only have few words (sentences);
:-1: @brytebee

Reply:
resolved 👍